src/main/java/org/jabref/logic/importer/fetcher/CitationFetcher.java

```diff
@@ -1,4 +1,4 @@
-package org.jabref.gui.entryeditor.citationrelationtab.semanticscholar;
+package org.jabref.logic.importer.fetcher;
```

Comment:
Move this to the logic package, `Fecthing` is more like a back-end process - should belong to an adapter layer.